### PR TITLE
Automatically include correct version in statistics script

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates the statistics script to automatically include the current version set in `package.json`.

### CODE REVIEW

File: scripts/statistics.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 const chalk = require('chalk');
 
 const bcd = require('..');
+const version = require('../package.json').version;
```
Comment (ddbeck) [inline scripts/statistics.js:9]: We can skip a require with `process.env.npm_package_version`:

```suggestion
```

File: scripts/statistics.js [hunk 2]
```diff
@@ -175,7 +176,7 @@ const printStats = (stats, folder) => {
   }
 
   console.log(
-    chalk`{bold Status as of version 1.0.xx (released on 2020-MM-DD) for ${
+    chalk`{bold Status as of version ${version} (released on 2020-MM-DD) for ${
```
Comment (ddbeck) [inline scripts/statistics.js:179]:
```suggestion
    chalk`{bold Status as of version ${process.env.npm_package_version} (released on 2020-MM-DD) for ${
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
One small suggestion in line comments. Thanks, @vinyldarkscratch!

File: scripts/statistics.js (hunk 1 above)
Comment (vinyldarkscratch) [reply scripts/statistics.js:9]: Ooh, I didn't know about that, I'll start using that in my other projects too!

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**